elements/noflo-icon-selector.html

```diff
@@ -0,0 +1,79 @@
+<link rel="import" href="noflo-icon.html">
+<dom-module id="noflo-icon-selector">
+  <template>
+    <style>
+    :host {
+      display: block;
+    }
+    ul.icons {
+      padding: 0px;
+      margin: 0px;
+      list-style: none;
+    }
+    ul.icons li {
+      display: inline-block;
+      width: 18px;
+      height: 18px;
+      font-size: 18px;
+      padding: 0px;
+      margin: 0px;
+      text-align: center;
+      overflow: hidden;
+      background-color: var(--noflo-ui-text);
+      color: var(--noflo-ui-background);
+      cursor: pointer;
+      transition: background-color 0.3s ease-in-out;
+      border-radius: 2px;
+      margin-right: 2px
+      margin-bottom: 2px;
+    }
+    ul.icons li.selected,
+    ul.icons li:hover {
+      background-color: hsl(190, 100%, 30%);
+      color: white;
+    }
+    </style>
+    <ul class="icons">
+      <template is="dom-repeat" items={{icons}} as="icon">
+        <li data-id$="[[icon]]" class$="[[_selectedClass(icon, selected)]]" on-click="selectIcon">
+          <noflo-icon icon="[[icon]]" title="[[icon]]"></noflo-icon>
+        </li>
+      </template>
+    </ul>
+  </template>
+  <script>
+  class NoFloIconSelector extends Polymer.Element {
+    static get is() {
+      return 'noflo-icon-selector';
+    }
+    static get properties() {
+      return {
+        icons: {
+          type: Array,
+          value: function () {
+            return [];
+          },
+        },
+        selected: {
```

Comment:
It is currently in the title, so you should be able to see it when hovering. But generally since icons _are_ visual, and only shown visually, the name should matter less than the visual representation

Reply:
Shown while hovering is OK.

The icons are not only shown visually. All edits except for the initial value needs to be done on the icon name string in the component source code.